system-int-test/test/cmr/system_int_test/ingest/collection_umm_spec_validation_test.clj

```diff
@@ -112,7 +112,7 @@
                            (data-umm-cmn/additional-attribute {:Name "bool2" :DataType "BOOLEAN" :Value true})]})
                        {:allow-failure? true})]
         (is (= {:status 422
-                :errors ["object has missing required properties ([\"CollectionProgress\",\"DataCenters\",\"Platforms\",\"ProcessingLevel\",\"ScienceKeywords\",\"TemporalExtents\"])"]}
```

Comment:
Yes it's required. but it's never gonna be empty with the new mapping.  Look at the ticket. If xml file doesn't contain the values, it'll be "NOT PROVIDED".